modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/localizable/text/LocalizableTextDDMFormFieldValueAccessorTest.java

```diff
@@ -51,6 +53,16 @@ public void testEmpty() {
 				DDMFormValuesTestUtil.createDDMFormFieldValue(
 					"localizableText", new UnlocalizedValue("{}")),
 				LocaleUtil.US));
+
+		LocalizedValue localizedValue = new LocalizedValue();
+
+		localizedValue.addString(LocaleUtil.US, StringPool.BLANK);
+
+		Assert.assertTrue(
+			_localizableTextDDMFormFieldValueAccessor.isEmpty(
+				DDMFormValuesTestUtil.createDDMFormFieldValue(
+					"localizableText", localizedValue),
+				LocaleUtil.US));
```

Comment:
Hi, thanks for working on this.

I think the right test for this use case would be something like:

```
localizedValue.addString(
	LocaleUtil.US,
	JSONUtil.put(
		"en_US", StringPool.BLANK
	).put(
		"pt_BR", StringPool.BLANK
	).toString());
```

The `LocalizableText` field is being treated as a `localizable` field, so its `localizedValue` (`{LanguageIdFromSite: value}`) is used during evaluation instead of just its `value` (the `LocalizedValuesMap`).

Reply:
Ah, good catch @carolmariaabb. You are correct, I reverted the fix and the current test still passed 😬 . I will apply your suggestion.